src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,66 @@
+package duke;
+
+import java.util.Scanner;
+
+/**
+ * Interacts with the console of Duke users, displaying operations, status, and errors.
+ */
+public class Ui {
+
+    /**
+     * Displays welcome message to users.
+     */
+    protected void intro() {
```

Comment:
`intro` doesn't seem suitable for a method name. Perhaps you could rename it to `greet`.
```suggestion
    protected void greet() {
```

Reply:
Thanks for all the pointers!